routes/web.php

```diff
@@ -0,0 +1,56 @@
+<?php
```

Comment:
You can group all the routes that need the user to be logged in by doing this as an example:
```
Route::group(['middleware' => [ 'auth'], function () {
Route::get('tasks/index',[ 'middleware' => 'App\Http\Middleware\RoleMiddleware:MANAGE_TASKS', 'uses' => 'App\Http\Controllers\TaskController@index']);
Route::get('tasks/create',[ 'middleware' => 'App\Http\Middleware\RoleMiddleware:MANAGE_TASKS', 'uses' => 'App\Http\Controllers\TaskController@create']);
Route::get('tasks/{id}/edit',[ 'middleware' => 'App\Http\Middleware\RoleMiddleware:MANAGE_TASKS', 'uses' => 'App\Http\Controllers\TaskController@edit']);
Route::get('tasks/{id}/assign',[ 'middleware' => 'App\Http\Middleware\RoleMiddleware:MANAGE_TASKS', 'uses' => 'App\Http\Controllers\TaskController@onAssign']);
Route::delete('tasks/{id}',[ 'middleware' => 'App\Http\Middleware\RoleMiddleware:MANAGE_TASKS', 'uses' => 'App\Http\Controllers\TaskController@delete']);
Route::post('tasks/store',[ 'middleware' => 'App\Http\Middleware\RoleMiddleware:MANAGE_TASKS', 'uses' => 'App\Http\Controllers\TaskController@store']);
Route::post('tasks/{id}/update',[ 'middleware' => 'App\Http\Middleware\RoleMiddleware:MANAGE_TASKS', 'uses' => 'App\Http\Controllers\TaskController@update']);
Route::post('tasks/{id}/assign',[ 'middleware' => 'App\Http\Middleware\RoleMiddleware:MANAGE_TASKS', 'uses' => 'App\Http\Controllers\TaskController@assign']);
Route::post('tasks/{id}/unassign',[ 'middleware' => 'App\Http\Middleware\RoleMiddleware:MANAGE_TASKS', 'uses' => 'App\Http\Controllers\TaskController@unassign']);
Route::post('tasks/{id}/complete',[ 'middleware' => 'App\Http\Middleware\RoleMiddleware:COMPLETE_TASKS', 'uses' => 'App\Http\Controllers\TaskController@complete']);
});
```

Reply:
done